boards/arm/wio_terminal/wio_terminal-pinctrl.dtsi

```diff
@@ -0,0 +1,82 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <dt-bindings/pinctrl/samd51p-pinctrl.h>
+
+&pinctrl {
+
+	/* RTL8720D (Wifi/BLE) */
```

Comment:
I think you want to move these type of comments to the .dts file instead. This here is just to describe the pinmuxes, and not making any assumptions on what peripherals etc are eventually going to be connected.

Reply:
The schematic do not offer many possibilities. Particularly for Wifi, LCD, SD Card etc. This is why I have added convenient comments here to remind the purpose of the pins.

Reply:
I was not suggesting to *not* comment on the purpose of each SERCOM, but rather saying that I think these comments should rather be in the wio_terminal.dts file, as this here is "purely" about how SERCOMs are configured, vs. making any assumption on what they may or may not be attached to. Plus, I would expect most "high-level" users would not even want to dig into wio_terminal-pinctrl.dtsi and would rather be interested in having the wio_terminal.dts file letting them know at a glance what SERCOM corresponds to what peripherals (especially for those peripherals that are not (yet) visible in the tree, such as the RTL chip)

Reply:
Done